build.gradle

```diff
@@ -9,8 +9,9 @@ group = 'nu.studer'
 version = '1.1-DEV'
 
 buildScan {
-    licenseAgreementUrl 'https://gradle.com/terms-of-service'
-    licenseAgree 'yes'
+    termsOfServiceUrl = 'https://gradle.com/terms-of-service'
+    termsOfServiceAgree = 'yes'
+    publishAlways()
```

Comment:
This should be removed. This should not be checked in for open source projects as it means that any user who builds this automatically publishes a build scan.

Reply:
Removed it again. Just did it as it was on the TeamCity project as well. Will remove it there as well.